addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -6,17 +6,17 @@ var UnsplashAPI = require('unsplash.api');
 var widgetsMedia = require('wysiwyg.widgets.media');
 
 var unsplashAPI = null;
+// Max number of database attachments to show when searching.
+const MAX_DB_ATTACHMENTS = 5;
```

Comment:
Can this be a static property of the media dialog so that it can be changed by a potential custo?